sql/src/main/java/io/crate/execution/dml/ShardResponse.java

```diff
@@ -160,6 +177,15 @@ public void writeTo(StreamOutput out) throws IOException {
         } else {
             out.writeBoolean(false);
         }
+
+        if (values != null) {
```

Comment:
The serialization can be optimized here, since the types of the rows are already known.